web/react/components/tutorial/tutorial_intro_screens.jsx

```diff
@@ -124,11 +125,34 @@ export default class TutorialIntroScreens extends React.Component {
                     {'.'}
                 </p>
                 {'Click “Next” to enter Town Square. This is the first channel teammates see when they sign up. Use it for posting updates everyone needs to know.'}
-                <div className='tutorial__circles'>
-                    <div className='circle'/>
-                    <div className='circle'/>
-                    <div className='circle active'/>
-                </div>
+                {circles}
+            </div>
+        );
+    }
+    createCircles() {
+        const circles = [];
+        for (let i = 0; i < NUM_SCREENS; i++) {
+            let className = 'circle';
+            if (i === this.state.currentScreen) {
+                className += ' active';
+            }
+
+            circles.push(
+                <a
+                    href='#'
+                    key={'circle' + i}
+                    className={className}
+                    onClick={(e) => { //eslint-disable-line no-loop-func
```

Comment:
This is probably a bug in eslint. It shouldn't warn here because your using let for `i` and not var.